### DISCUSSION TITLE
Automatic clipboard sharing also possible?

### DISCUSSION THREAD

**Discussion by Greatz08** [lima-vm/lima#2443] [Q&A]
?

**Comment by afbjorklund** [comment]
Do you have some more details, on what clipboard and where it should be shared?

* https://github.com/lima-vm/lima/issues/424

**Comment by andy0130tw** [comment]
I guess OP means the clipboard integration `spice-vdagent` provides, as shown in the [Spice and Spice-related components](https://www.spice-space.org/spice-user-manual.html#agent) section:

> ## Spice Agent
> 
> The Spice agent is an optional component for enhancing user experience and performing guest-oriented management tasks. [...] Other features of agent are shared clipboard (copy and paste between guest and host) and aligning guest resolution with client when entering fullscreen mode.
